packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -29,6 +31,34 @@ New usage:
   }
 ```
 
+### Applications using a child design system
+
+The `core`, `support`, `layout` npm packages are no longer needed for applications using a child design system. The only dependency needed is the child design system package in your `package.json`.
```

Comment:
It would also be nice to link to the new child ds page: `{{root}}/startup/child-design-systems/`